src/Api/JsonRpc/Repository/TestExecutionRepository.php

```diff
@@ -64,7 +64,7 @@ public function create(TestExecution $model): TestExecution
             'run_id' => $modelData['run']
         ]));
 
-        $result = $response->getRpcResult();
+        $result = $response->getRpcResult()[0];
```

Comment:
Note: since v11 this method returns a list and if the test case has parameters (we call them properties) then multiple executions need to be created. Can you open an issue for this to be fixed later? The approach in other plugins is that they iterate over the results.